config/fcos/v1_6_exp/validate.go

```diff
@@ -15,13 +15,30 @@
 package v1_6_exp
 
 import (
+	"regexp"
+
 	"github.com/coreos/butane/config/common"
 	"github.com/coreos/ignition/v2/config/util"
 
 	"github.com/coreos/vcontext/path"
 	"github.com/coreos/vcontext/report"
 )
 
+var regex = regexp.MustCompile(`^/(etc|var)/.*`)
+
+func (conf Config) Validate(c path.ContextPath) (r report.Report) {
+	for _, fs := range conf.Storage.Filesystems {
+		if util.NilOrEmpty(fs.Path) {
+			r.AddOnError(c.Append("storage"), common.ErrMountPointForbidden)
```

Comment:
I dont like having to have dupe code like these two "addOnErrror". I think we should step back to how it used to be. Sorry for the extra steps.

Reply:
no problem have it changed back now!